dawg_python/wrapper.py

```diff
@@ -17,29 +17,29 @@ def __init__(self):
     "Root index"
 
     def has_value(self, index):
-        "Checks if a given index is related to the end of a key."
+        #Checks if a given index is related to the end of a key.
```

Comment:
The code currently isn't pep8 compatible anyway because some older lines go over 80 chars, etc... I'm thinking let's leave this as is for now, and I'll do a later run through to get everything pep8 compatible so the whole repo passes pep8 compatibility checks.